spec/components/bulk_actions_form_component_spec.rb

```diff
@@ -6,12 +6,17 @@
   subject(:instance) { described_class.new(form: bulk_action_form) }
 
   let(:bulk_action_form) { BulkActionForm.new(build(:bulk_action), groups: []) }
+  let(:blacklight_config) { Blacklight::Configuration.new }
+  let(:search_state) { Blacklight::SearchState.new(search_params, blacklight_config) }
 
-  before { allow(controller).to receive(:current_user).and_return(build(:user)) }
+  before do
+    allow(controller).to receive(:current_user).and_return(build(:user))
+    allow(subject).to receive(:search_state).and_return(search_state)
+  end
 
   describe '#search_of_pids' do
     context "when last search isn't present" do
```

Comment:
I don't think this is a meaningful test anymore (it's the same as the next one)

Reply:
removed